src/main/java/happy_family/Main.java

```diff
@@ -64,5 +66,38 @@ public static void main(String[] args) {
         familyService.fillWithTestData();
         FamilyController familyController = new FamilyController(familyService);
         familyController.start();
+
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.println("\nMenu:");
+            System.out.println("1. Display all families");
+            System.out.println("2. Save data to file");
+            System.out.println("3. Load data from file");
+            System.out.println("4. Exit");
+
+            int choice = scanner.nextInt();
+            scanner.nextLine();  // consume newline
+
+            switch (choice) {
+                case 1:
+                    familyService.getAllFamilies().forEach(System.out::println);
+                    break;
+                case 2:
+                    System.out.print("Enter file path to save data: ");
+                    String saveFilePath = scanner.nextLine();
+                    familyService.saveDataToFile(saveFilePath);
+                    break;
+                case 3:
+                    System.out.print("Enter file path to load data: ");
+                    String loadFilePath = scanner.nextLine();
+                    familyService.loadDataFromFile(loadFilePath);
```

Comment:
it will be good hold you file path inside your code, don't get file path from user